react/components/organisms/profile/profile.js

```diff
@@ -12,6 +12,7 @@ export class Profile extends Component {
         return {
             account: PropTypes.object.isRequired,
             actions: PropTypes.object,
+            buttons: PropTypes.array,

```

Comment:
Do we want to replace the entirety of the buttons? This is for localization later right? Why not, similar to the other components above, just have a shareContactText and signOutText?